interface-definitions/nat_cgnat.xml.in

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="nat">
+    <children>
+      <node name="cgnat" owner="${vyos_conf_scripts_dir}/nat_cgnat.py">
+        <properties>
+          <help>Carrier-grade NAT (CGNAT) parameters</help>
+          <priority>221</priority>
+        </properties>
+        <children>
+          <node name="pool">
+            <properties>
+              <help>External and internal pool parameters</help>
+            </properties>
+            <children>
+              <tagNode name="external">
+                <properties>
+                  <help>External pool name</help>
+                </properties>
+                <children>
+                  <leafNode name="external-port-range">
+                    <properties>
+                      <help>Port range</help>
+                      <valueHelp>
+                        <format>range</format>
+                        <description>Numbered port range (e.g., 1001-1005)</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="port-range"/>
+                      </constraint>
+                    </properties>
+                    <defaultValue>1024-65535</defaultValue>
+                  </leafNode>
+                  <node name="per-user-limit">
+                    <properties>
+                      <help>Per user limits for the pool</help>
+                    </properties>
+                    <children>
+                      <leafNode name="port">
```

Comment:
Please use `port-number.xml.i` building block

Reply:
The `port-number.xml.i` uses general help `Port number used by connection`.
It is completely incorrect, as we set the number of ports that will be allocated per user.
It is the reason for using a new leafNode port